### ISSUE TITLE
[IMPROVEMENT] Informationen zum Ingamesupport

### ISSUE THREAD

**Issue by motrellin** [FLBadoras/wiki#107]
### Grobbeschreibung

Support-Seite, Ingamesupport

### Beschreibung

Auf der [Support-Seite unter "Kontaktmöglichkeiten"](https://wiki.badoras.life/support#punktesystem) fehlen noch Informationen zum Ingame-Support.

**Comment by motrellin** [comment]
@Ru8ior das war ja einfach ein Community-Support, den man mit /h aufrufen kann, oder? Ergänze ich dann bei Gelegenheit selbst, wenn ich dir nicht zu langsam bin.

Related to #22

**Comment by Ru8ior** [comment]
Genau, es gibt kein Befehl mehr für irgendwelche Tickets, sondern ganz einfach /h, kannst du @motrellin ansonsten gerne einfach ergänzen.
